app/src/androidTest/java/com/github/studydistractor/sdp/MainNavigationTest.kt

```diff
@@ -1,32 +1,35 @@
 package com.github.studydistractor.sdp
 
+import androidx.compose.runtime.snapshots.SnapshotStateList
 import androidx.compose.ui.test.assertIsDisplayed
+import androidx.compose.ui.test.junit4.createAndroidComposeRule
 import androidx.compose.ui.test.junit4.createComposeRule
 import androidx.compose.ui.test.onNodeWithTag
 import androidx.compose.ui.test.performClick
 import androidx.ui.core.compose
+import com.github.studydistractor.sdp.history.HistoryEntry
+import com.github.studydistractor.sdp.history.HistoryInterface
+import com.github.studydistractor.sdp.procrastinationActivity.ProcrastinationActivityService
 import com.github.studydistractor.sdp.ui.RegisterScreen
 import dagger.hilt.android.testing.HiltAndroidRule
 import dagger.hilt.android.testing.HiltAndroidTest
 import org.junit.Before
 import org.junit.Rule
 import org.junit.Test
+import javax.inject.Inject
 
 @HiltAndroidTest
 class MainNavigationTest {
     @get:Rule(order = 0)
     var rule = HiltAndroidRule(this)
 
     @get:Rule(order = 1)
-    val composeRule = createComposeRule()
+    val composeRule = createAndroidComposeRule<MainActivity>()

```

Comment:
What is the purpose of this change?

Reply:
This is not one of my changes, but coming from the commit I wrote my code on top of.